ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -1,119 +1,222 @@
 #!nsh
 #
-# Standard startup script for sensor drivers.
+# Sensor driver startup script.
 #
+# NOTE: environment variable references:
+#    If the dollar sign ('$') is followed by a left bracket ('{') then the
+#    variable name is terminated with the right bracket character ('}').
+#    Otherwise, the variable name goes to the end of the argument.
+#
+#
+# NOTE: COMMENT LINES ARE REMOVED BEFORE STORED IN ROMFS.
+#
+
+#
+# Declare local script variables and initialize default values.
+#
+set BOARD_FMUV3    false
+
+#
+# Setup for generic configurations.
+#
+
+# Configure all I2C buses to 100 KHz as they are all external or slow.
+fmu i2c 1 100000
+fmu i2c 2 100000
+
+# ADC.
+adc start
+
+# STMicroelectronics 9-DOF IMU/Magnetometer.
+mpu9250 -s -R 14 start
```

Comment:
This was not started before unconditionally. What is your reason for adding it?
Similar the ones below.

Reply:
@bkueng, reverting that code does cause breakage in the current construct as pushed up to this PR at the time of writing this comment.  The logic is tangled and I am still attempting to de-tangle and simplify the script.  I will require a short amount of additional time to troubleshoot this set of details and report back.  Thank you again for your feedback!

Reply:
Ah... @bkueng, those script lines would have occurred as default as long as the hwcmp was not AEROFC_V1 or OMNIBUS_F4SD... this is what made them seem to be acceptable as generic cases.  Please advise me on how you would like this to be handled.

Reply:
@bkueng, I went ahead and relocated those driver start lines to each of the other vehicle cases that were utilizing them.  It duplicates those startups across vehicle cases, but I think it is the easiest way to kep future hardware specific logic from growing the number of if/then/else statements as has been the trend.  In this manner, in the future, if another driver is added for any specific board it should be very easy to not affect any other vehicle configuration.